### DISCUSSION TITLE
How to delete all bookmarks and folders

### DISCUSSION THREAD

**Discussion by Antergosgeek** [nextcloud/bookmarks#2172] [Q&A] [answered]
Hello,

I somehow can not figure out how to mass delete bookmarks and folders. I am using nextcloud on docker and installed the bookmarks addon and have 100s of bookmarks and multiple nested folders that I would like to delete.

Can anyone help?

**Comment by marcelklehr** [comment, marked as answer]
Hi @Antergosgeek 

You can select a bookmark, then in the selection menu click on "Select all" and "Delete". Then delete all folders.

**Comment by Antergosgeek** [comment]
Hi @marcelklehr ,

Thank you so much! No I feel a little embarrassed, never occurred to me that it could give that option once I select a bookmark...
Topic solved. Is there a wiki somewhere? I could not find anything that is why I reached out here.

**Reply by marcelklehr** [reply]
No worries! There is a wiki over here: https://github.com/nextcloud/bookmarks/wiki But it hasn't been used much so far. Maybe you can start an FAQ article? :)

**Comment by Antergosgeek** [comment]
@marcelklehr 

I'd be glad to. Do I need any sort of permissions to do so?
